src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Lottery loteria = new Lottery();
```

Comment:
The variable name 'loteria' should be in English and more descriptive. Consider renaming it to 'lottery' to match the class name and improve code readability.